### PR TITLE
Limit nodes to local network.

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

**Comment by afck** [comment]
…because somehow the tests manage to make my DSL connection unstable. (I know, it doesn't really make sense. :alien: :ghost:)

But they are meant to be purely local anyway, so it shouldn't hurt to disable other connections?

File: scripts/start-test-setup [hunk 1]
```diff
@@ -8,6 +8,6 @@ NODE=`which node`
 $NODE ./scripts/addCandidateBalances.js
 for i in $(seq 0 6); do
     rm -rf -- "./parity-data/node$i/chains"
-    "$PARITY" --config "./config/node${i}.toml" > "./parity-data/node${i}/log" 2>&1 &
+    "$PARITY" --interface=local --nat=none --config "./config/node${i}.toml" > "./parity-data/node${i}/log" 2>&1 &
```
Comment (vkomenda) [inline scripts/start-test-setup]: Do you need the `--nat` option given that we have it in the `toml` files? https://github.com/poanetwork/posdao-test-setup/blob/e003aed8621fea3fd1441cf3689d911bf4973e85/config/node0.toml#L11

Comment (afck) [reply scripts/start-test-setup]: You're right; I removed both from the script and added `network = "local"` to the config files.

**Comment by afck** [comment]
Sorry! I had messed that up when moving from the script to the config files. Fixed now.

**Comment by lazaridiscom** [comment]
> …because somehow the tests manage to make my DSL connection unstable. (I know, it doesn't really make sense.

Confirmed, I had this too.

**Pull request merged**